static/app/views/replays/deadClickList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as Layout from 'sentry/components/layouts/thirds';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function DeadClickList() {
+  const organization = useOrganization();
+  const hasDeadCicks = organization.features.includes(
```

Comment:
do we need a feature flag here?

Reply:
yup, for a brand new feature/area of the app (like feedback, or when replays was new) we might look at putting the feature check inside of `static/app/views/replays/index.tsx` or something, but for this, since we're guarding only the one page, this looks like the perfect spot.